src/controller/CHIPDeviceController.h

```diff
@@ -548,6 +601,13 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController, public SessionEst
 
     DeviceCommissionerRendezvousAdvertisementDelegate mRendezvousAdvDelegate;
 
+#if CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY // make this commissioner discoverable
+    UserDirectedCommissioningServer * mUdcServer = nullptr;
+    // mUdcTransportMgr is for insecure communication (ex. user directed commissioning)
+    DeviceTransportMgr * mUdcTransportMgr = nullptr;
+    uint16_t mUdcListenPort               = CHIP_PORT + 1;
```

Comment:
Once we have commissioner discovery with advertisement of the `_matterd._udp`service type, we should move UDC to an ephemeral port in the core sdk and examples.

Reply:
The actual port used is set using the "SetUdcListenPort(uint16_t listenPort)" method of DeviceCommissioner which follows the convention used by DeviceController which has its analogous "SetUdpListenPort(uint16_t listenPort)". In both cases, the caller decides what actual port to use.